internal/controller/eventbus/eventlog/eventlog.go

```diff
@@ -888,7 +888,7 @@ func (el *eventlog) add(ctx context.Context, seg *Segment) error {
 	el.mutex.Lock()
 	defer el.mutex.Unlock()
 	if !seg.isReady() {
-		return errors.ErrInvalidSegment
+		return errors.ErrServerNotRunning.WithMessage("the segment not ready")
```

Comment:
why change error type?

Reply:
I think the segment is not ready, which can be described as ServerNotRunning, to simplify the error code